lib/block-patterns.php

```diff
@@ -204,7 +204,7 @@ function remove_core_patterns() {
  *
  * @since 5.8.0
  */
-function load_remote_patterns() {
+function load_remote_core_patterns() {
```

Comment:
Is there a reason we're not prefixing these functions?

Reply:
It seems not - I added the prefix. I guess we should also loose the `_` in core(https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/block-patterns.php#L54), no? --cc @ryelle